neuralforecast/common/_base_windows.py

```diff
@@ -234,9 +245,17 @@ def _create_windows(self, batch, step):
 
         elif step in ["predict", "val"]:
             if step == "predict":
+                initial_input = temporal.shape[-1] - self.test_size
+                if (
+                    initial_input <= self.input_size
+                ):  # There is not enough data to predict first timestamp
+                    padder_left = nn.ConstantPad1d(
```

Comment:
Ibid 
Comment on missing work to add encoded static features into the initial condition of the encoders.

Reply:
I prefer to add pending work on issues instead of directly in the code. But just a preference.